src/map/lua/lua_baseentity.cpp

```diff
@@ -2089,17 +2089,10 @@ bool CLuaBaseEntity::sendGuild(uint16 guildID, uint8 open, uint8 close, uint8 ho
     XI_DEBUG_BREAK_IF(open > close);
 
     uint8 VanadielHour = (uint8)CVanaTime::getInstance()->getHour();
-    // uint8 VanadielDay = (uint8)CVanaTime::getInstance()->getWeekday();
 
     GUILDSTATUS status = GUILD_OPEN;
 
-    /*
-     * No more guild holidays since 2014
-    if (VanadielDay == holiday)
-    {
-        status = GUILD_HOLYDAY;
```

Comment:
on the fence about this commented out block - it's possible it's been saving us from having to explain how to do it with every new "oldschool" server that pops up :laughing:

Reply:
Makes sense.  Restored the information that was removed in that function